data/map.txt

```diff
@@ -27944,6 +27964,15 @@ planet Furnace
 		threshold 3500
 		fleet "Large Syndicate" 7
 
+planet "Garden Empyreal"
+	attributes kimek "coalition station" shipping wealthy tourism
+	landscape land/clouds_03
+	description `Not counting the trio of Quarg ringworlds, this is the largest megastructure in Coalition space: an array of gigantic metal pyramids, each one nearly a dozen times larger than the average space station. Spread out over the planet's upper atmosphere, these so-called "hubs" are connected by long, inextensible, and extremely sturdy wires, ensuring no part of the structure is destabilized.`
+	description `	First devised by the Kimek before the Coalition's war with the Quarg, the structure was actually completed relatively recently, just two thousand years ago. Populating the gas giant's uppermost level, several hardy species of artificially engineered bacteria are cultivated, being periodically "scooped" by the lower body of each hub, and mass-produced into a nutritious soup.`
+	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`
```

Comment:
"floating silver palaces" is supposed to mean the pyramid hubs.

Reply:
Ah, ok...
Might want to address it so that it makes the connection a bit more obvious, something like referencing the triangle-ness of  each hub, or reflection, or something close to that.

Reply:
Better. Yep.